packages/luna/src/components/TutorFeedback/TutorRatingCardGroup.tsx

```diff
@@ -83,7 +83,7 @@ export const TutorRatingCardGroup: React.FC<TutorRatingCardGroupProps> = ({
                   <Avatar
                     alt={orUndefined(rating.name)}
                     seed={rating.userId.toString()}
-                    src={idx <= 4 ? orUndefined(rating.imageUrl) : undefined}
+                    src={idx <= 4 ? orUndefined(rating.image) : undefined}
```

Comment:
We prefare to use the name image url to indicate the this the full image url and not the image name.